tests/test-frameworks/isolation/test

```diff
@@ -13,3 +13,7 @@ bazel build :classloader-shared
 ! bazel test :classloader-shared
 
 bazel test :process :process-2_11 :process-jvm
+
+bazel test --test_arg=--framework_args=-showtimes :none | grep 'be 2.*ms'
```

Comment:
looks great! Could you also add a negative test?
Something simple like: `! (bazel test :none | grep 'be 2.*ms')`